apis/field_error.go

```diff
@@ -84,6 +84,20 @@ func (fe *FieldError) ViaIndex(index int) *FieldError {
 	return fe.ViaField(asIndex(index))
 }
 
+// AtIndex attaches the index to the current field.
+// Which is different from ViaIndex, that attaches index to the
+// next field.
+func (fe *FieldError) AtIndex(index int) *FieldError {
```

Comment:
how is this different than `ViaIndex`?

Reply:
I think the docs make it pretty clear, but @vagababov was running into a problem indicating an error with the second item in a `[]string`, i.e. `releases[1]`. This ends up being the syntax for that:

```golang
 errs = errs.Also(apis.ErrInvalidValue(r, apis.CurrentField).ViaFieldIndex("revisions", i))
```